workflows/theiacov/wf_theiacov_clearlabs.wdl

```diff
@@ -121,6 +124,30 @@ workflow theiacov_clearlabs {
         genome_fasta = consensus.consensus_seq,
         assembly_length_unambiguous = consensus_qc.number_ATCG
     }
+  if(defined(qc_check_table)) {
+    call qc_check.qc_check_phb as qc_check_task {
+      input:
+        qc_check_table = qc_check_table,
+        expected_taxon = organism,
```

Comment:
Does this need to be set as a task input for TheiaCoV workflows as well? Based on my understanding of th qc_check_phb task, it looks as if this will only be relevant to bacterial samples being typed by Gambit.

Reply:
Ah, never mind! I see what's happening: This sets the `qc_taxon` variable based on _either_ `expected_taxon` (provided via `organism` in TheiaCoV workflows) or by `gambit_predicted_taxon` (provided via the gambit task TheiaProk).